vidarr-pluginapi/src/main/java/ca/on/oicr/gsi/vidarr/ConsumableResource.java

```diff
@@ -77,10 +77,10 @@ public JavaType typeFromId(DatabindContext context, String id) throws IOExceptio
    * @param workflowName the name of the workflow
    * @param workflowVersion the version of the workflow
    * @param vidarrId the identifier of the workflow run
-   * @param isLaunched a boolean flag specifying whether run is launched
+   * @param isComplete a boolean flag specifying whether run is launched
```

Comment:
I'm sorry, but, this javadoc needs to be updated again

Reply:
Good catch!